Auth/Strasnote.Auth.Api/Startup.cs

```diff
@@ -55,7 +55,8 @@ public void ConfigureServices(IServiceCollection services)
 					{
 						ValidIssuer = "https://localhost:5001",
 						ValidAudience = "https://localhost:5001",
-						IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("Fanatic-Onion8-Sports")),
+						// ToDo: pull out of secrets. what's the best way to do this in ConfigureServices()?
+						IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("Fanatic-Onion8-Sports")), 
```

Comment:
These are already all in appsettings - you just need to make appsettings.Secrets.json with it in and the app will load it

Reply:
We don't have the usual method of access into IConfiguration from here

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-5.0#accessing-options-during-startup

> Don't use IOptions<TOptions> or IOptionsMonitor<TOptions> in Startup.ConfigureServices. An inconsistent options state may exist due to the ordering of service registrations.